includes/admin.form.inc

```diff
@@ -24,6 +24,12 @@ function islandora_bookmark_admin_settings(array $form, array &$form_state) {
       '#description' => t('The unpluralized name of the type of lists such as bookmark or favorite.'),
       '#default_value' => variable_get('islandora_bookmark_type', 'bookmark'),
     ),
+    'anonymous_owner' => array(
+      '#type' => 'textfield',
+      '#title' => t('The owner name to display for anonymous users.'),
+      '#description' => t('The name to display in the owner column for users who are not logged in.'),
+      '#default_value' => variable_get('islandora_bookmark_owner', 'Me'),
```

Comment:
@J77D,

This is definitely a great additive change but to ensure existing functionality it'd be preferable to have the default value of the `variable_get` be equivalent to the old, hardcoded value (Anonymous).

Thanks!
----
Jordan Dukart
_Developer_
*[discoverygarden inc.|http://www.discoverygarden.ca] | Managing Digital Content*